Tests/PrefixDurationTests.swift

```diff
@@ -47,7 +47,7 @@ final class PrefixDurationTests: XCTestCase {
         var completions = [Subscribers.Completion<Never>]()
 
         cancellable = subject
-            .prefix(duration: 0.5)
+            .prefix(duration: 0.8)
```

Comment:
Sneaky you 
<img src="https://media.giphy.com/media/xT39DkkGBwez9AXHX2/giphy.gif" />